core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Run.java

```diff
@@ -48,241 +98,550 @@
  * @author Rob Davey
  * @since 0.0.2
  */
-@JsonSerialize(typing = JsonSerialize.Typing.STATIC)
-@JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "@class")
-@JsonIgnoreProperties({ "securityProfile", "submissionDocument" })
-public interface Run
```

Comment:
I can agree that a lot of the model interfaces are not useful - the simpler ones mainly. I don't see the benefit to removing interfaces that already exist though, as removing them requires no less effort than maintaining them does, and we do find alternate uses for some of them.

For this specific case, a RunBuilder that implements IlluminaRun, PacBioRun, SolidRun, etc. would be useful for simplifying the UI. Without it, the run type would have to be specified in the request, which means extra clutter, clicking, and page loads on the UI. It's not a big deal, but it's also easy to go the better route.

If we want to go the worse route, or there's an alternate solution, that's fine too